src/fromager/request_session.py

```diff
@@ -0,0 +1,7 @@
+import requests
+from requests.adapters import HTTPAdapter
+
+session = requests.Session()
+adapter = HTTPAdapter(max_retries=5)
```

Comment:
Does adapter need to be public, or could it be renamed `_adapter`?

Reply:
yeah, renamed to `_adapter`